packages/frontend/src/keybindings.ts

```diff
@@ -154,7 +157,10 @@ export function keyDownEvent2Action(
       if ((ev.target as HTMLElement)?.id === 'composer-textarea') {
         return KeybindAction.MessageList_PageDown
       }
-    } else if ((ev.metaKey || ev.ctrlKey) && ev.key === '/') {
+    } else if (
+      ((ev.metaKey || ev.ctrlKey) && ev.key === '/') ||
+      (!isLatin && ev.code === 'KeySlash')
```

Comment:
Have you tested this? Looks like this key does not exist.

```suggestion
      (!isLatin && ev.code === 'Slash')
```